_server_execute_js.js

```diff
@@ -7,26 +7,36 @@ module.exports = function() {
     */
     hybridVersion: function() {
       const ipcRenderer = require('electron').ipcRenderer;
+      const remote = require('electron').remote;
+      let pc = remote.getGlobal('pageCache');
+      let cachedValue = pc.get('_true_' + document.documentElement.outerHTML);
 
-      if (document.querySelectorAll('link[rel="import"]').length > 0) {
-        let html = document.cloneNode(true);
-        html.querySelector('body').removeAttribute('unresolved');
+      if(cachedValue == -1) {
```

Comment:
`cachedValue === -1`

The `===` in JavaScript makes sure the value is exactly what you want, whereas the `==` is a little fudgey.  (For example `"-1" == -1` and `[-1] == -1`)